deepchem/feat/graph_data.py

```diff
@@ -90,7 +90,26 @@ def __init__(
     self.num_edges = edge_index.shape[1]
     if self.edge_features is not None:
       self.num_edge_features = self.edge_features.shape[1]
+  
+  def __repr__(self):
+    """Utility to print the graph in more useful way.
+
```

Comment:
Probably we don't need an example here. Instead you can combine it with this example https://github.com/deepchem/deepchem/blob/059ffa00f3e22e23447f99e11d911e0e6e8d5c5d/deepchem/feat/graph_data.py#L35